Things_You_Should_Know/things_you_should_know.md

```diff
@@ -0,0 +1,400 @@
+# Things you should know: Web Apps on Linux
+
+**App Service** on Linux supports 2 scenarios:
+
+- Bring your own **CODE**
+- Bring your own **CONTAINER**
+
+For **CODE** scenarios, **App Service** provides a base container that is maintained by the platform. This container targets a specific development framework (.NET Core, PHP, Node, etc...) and a specific version of that framework (.NET Core 3.0, .NET Core 3.1, etc...).
+
+For **CONTAINER** scenarios, **App Service** provides a host where a custom container provided by the customer can execute.
+
+In both scenarios, a set of **App Service** features are exposed through the ARM API as well as through the Azure Portal and Azure CLI / Powershell experience.
+
+## General Information
+
+### You can requests features to the App Service team
+
+|**CODE**|**CONTAINER**|
+|:------:|:-----------:|
+| ✅    | ✅          |
+
+**App Service** has a [Feedback forum](http://aka.ms/webapps-uservoice), here you can request new features and voted on them for priority. Please make sure to put "[Linux]" in the title so that we'll know it applies to **App Service** on Linux.
+
+## Deployment
+
+### Web Deploy
```

Comment:
I would remove this section - Web Deploy to Linux App Service isn't really a deployment model that we want to be pushing.